src/app.ts

```diff
@@ -1,9 +1,12 @@
-import Vue from 'vue/dist/vue';
-import 'materialize-css/dist/js/materialize.min';
+import Vue from 'vue/dist/vue.esm';
+import prepareVuetify from './plugins/vuetify.ts';
```

Comment:
We can write @/plugins/vuetiify.ts => @ in Vue is an alias for src
It may be a good idea to keep absolute path, don't you think ?

Reply:
This doesn't seams to work...

Reply:
My bad. It seems, I don't really get it how it works to have '@' as an alias. I will check this to find how it works.